cspell/custom-dict.txt

```diff
@@ -81,3 +81,9 @@ xdg
 xdist
 xsser
 zsc
+bsky
+slideshare
+Slideshare
```

Comment:
Do we really need to have these repeated? I believe our spell checked is case insensitive
https://github.com/OWASP/Nest/blob/3e41cd84a5133d3fbe0febc874db8b307c7846e0/cspell/cspell.json#L3

Reply:
Same for Tiktok.

Reply:
My bad,  i removed those in latest commit.

Reply:
@jaiakash Thank you! 
And just one note for any future PRs - please do not resolve comments left in PRs. The PR reviewer should resolve those - it helps to see if the issue was addressed. Thanks!